extensions/applicationinsights-analytics-js/src/JavaScriptSDK/AnalyticsPlugin.ts

```diff
@@ -627,6 +622,16 @@ export class AnalyticsPlugin extends BaseTelemetryPlugin implements IAppInsights
                 return [_errorHookCnt, _autoExceptionInstrumented];
             };
             
+            function _resetAjaxAttempts() {
+                // Reset ajax attempts counter for the new page view
+                if (_self.core) {
+                    let ajaxPlugin = _self.core.getPlugin<IDependenciesPlugin>("AjaxDependencyPlugin");
+                    if (ajaxPlugin && ajaxPlugin.resetAjaxAttempts) {
```

Comment:
ajaxPlugin returns an ILoadedPlugin instance you need to 
- check that ajaxPlugin.plugin is exists
- check that ajaxPlugin.plugin.resetAjaxAttempts exists
- call the function via the .plugin reference